// [[Reason: DTE preserves different duplicate identifier]] ////

//// [tests/cases/compiler/duplicateObjectLiteralProperty_computedName1.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -17,9 +17,9 @@
 declare const t5: {
     "+1": number;
 };
 declare const t6: {
-    [-1]: number;
+    "-1": number;
 };
 declare const t7: {
     "-1": number;
 };
